### PR TITLE
Builds gsl-config binary ampl/gsl#11.

### PR DESCRIPTION
**Pull request by dilawar**

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -12,6 +12,9 @@ if (POLICY CMP0054)
   # Only interpret `if` arguments as variables or keywords when unquoted.
   cmake_policy(SET CMP0054 NEW)
 endif ()
+if( POLICY CMP0053 )
+    cmake_policy(SET CMP0053 NEW)
+endif( )
```
Comment (vitaut) [inline CMakeLists.txt]: nit: here and below, please follow the coding style of this `CMakeLists.txt`, in particular avoid extra spaces within parentheses and use 2-space indent:

```cmake
if (POLICY CMP0053)
  cmake_policy(SET CMP0053 NEW)
endif ()
```

File: CMakeLists.txt [hunk 2]
```diff
@@ -692,4 +695,27 @@ if (GSL_INSTALL OR NOT DEFINED GSL_INSTALL)
   set(PC_FILE ${CMAKE_BINARY_DIR}/gsl.pc)
   configure_file("gsl.pc.cmake" ${PC_FILE} @ONLY)
   install(FILES ${PC_FILE} DESTINATION lib/pkgconfig)
+
+  # Read config file and replace placefolders.
```
Comment (vitaut) [inline CMakeLists.txt]: typo: placefolders

File: CMakeLists.txt [hunk 3]
```diff
@@ -692,4 +695,27 @@ if (GSL_INSTALL OR NOT DEFINED GSL_INSTALL)
...
+  string( REPLACE @GSL_LIBS@ "-lgsl -lgslcblas" GSL_CONFIG_FILE
+      "${GSL_CONFIG_FILE}")
+  string( REPLACE @GSL_LIBM@ "-lm" GSL_CONFIG_FILE "${GSL_CONFIG_FILE}")
+
+  # write
+  string(REPLACE "@SEMICOLON@"  "\\;" GSL_CONFIG_FILE "${GSL_CONFIG_FILE}")
+  file( WRITE ${CMAKE_BINARY_DIR}/gsl-config ${GSL_CONFIG_FILE})
+
+  # Install gsl-config
+  install(PROGRAMS ${CMAKE_BINARY_DIR}/gsl-config DESTINATION bin)
```
Comment (vitaut) [inline CMakeLists.txt]: Why is `gsl-config` installed in the bin directory? Is it a script?

Comment (dilawar) [inline CMakeLists.txt]: Yes. its a bash script generated by gsl build system. E.g. on my system, after install gsl development package:

```
which gsl-config
/usr/bin/gsl-config
```

```
gsl-config 
Usage: gsl-config [OPTION]

Known values for OPTION are:

  --prefix		show GSL installation prefix 
  --libs		print library linking information, with cblas
  --libs-without-cblas	print library linking information, without cblas
  --cflags		print pre-processor and compiler flags
  --help		display this help and exit
  --version		output version information

An external CBLAS library can be specified using the GSL_CBLAS_LIB
environment variable. The GSL CBLAS library is used by default.

````

**Comment by vitaut** [comment]
Merged, thanks!

**Pull request merged**